manifests/monitor/tidb-monitor.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: pingcap.com/v1alpha1
```

Comment:
these is already an example of TidbMonitor: `examples/basic/tidb-monitor.yaml`

Reply:
Those are basic examples and users cannot know how to use the CR with the basic ones.